packages/module/src/CullingInfo/CullingInfoUtils.ts

```diff
@@ -0,0 +1,32 @@
+export type CullingDate = string | number | Date;
+
+export interface CullingInfo {
+  isWarn?: boolean;
+  isError?: boolean;
+  msg: string;
+}
+export type CalculateTooltip = (culled: CullingDate, warning: CullingDate, currDate: CullingDate) => CullingInfo;
+
+export const seconds = 1000;
+export const minutes: number = seconds * 60;
+export const hours: number = minutes * 60;
+export const days: number = hours * 24;
+
+export const calculateTooltip: CalculateTooltip = (culled, warning, currDate) => {
+  const culledDate: Date = new Date(culled);
+  const warningDate: Date = new Date(warning);
+  const diffTime: number = new Date(currDate).valueOf() - warningDate.valueOf();
+  const removeIn: number = Math.ceil((culledDate.valueOf() - new Date(currDate).valueOf()) / days);
+  const msg = `System scheduled for inventory removal in ${removeIn} days`;
```

Comment:
it might be better to put something more generic there and allow customizing the text.  In this case, I would probably also move this to the component's file and get rid of the utils file